### PR TITLE
Fix incorrect Redis URL snippets in the example app.ini

### LINKED ISSUE #22571: Example App config has incorrect example Redis strings
### Description

In the redis connection string for the queue, session, and task configuration the example config in `custom/conf/app.example.ini` suggests using a connection string formatted like `network=tcp,addr=:6379,password=macaron,db=0,pool_size=100,idle_timeout=180` but gitea expects a uri like connection string like this `reddis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`. If you follow the example in `custom/conf/app.example.ini` it causes gitea to throw errors or fail to start and does not match the [config cheat sheet](https://docs.gitea.io/en-us/config-cheat-sheet/#cache-cache) in the documentation.

### Gitea Version

1.18.2

### Can you reproduce the bug on the Gitea demo site?

No

### Log Gist

_No response_

### Screenshots

_No response_

### Git Version

_No response_

### Operating System

OCI Image on Fedora 37

### How are you running Gitea?

I am running gitea in via podman compose on fedora 37 behind a traefik reverse along side ansible-semaphore and woodpecker ci. 

### Database

MySQL

### PR DESCRIPTION
**Pull request by tiny-pangolin**
Fixes #22571

### CODE REVIEW

File: custom/conf/app.example.ini [hunk 1]
```diff
@@ -1721,7 +1721,7 @@ ROUTER = console
 ;INTERVAL = 60
 ;;
 ;; For "redis" and "memcache", connection host address
-;; redis: network=tcp,addr=:6379,password=macaron,db=0,pool_size=100,idle_timeout=180
+;; redis: reddis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s
```
Comment (yardenshoham) [inline custom/conf/app.example.ini]:
```suggestion
;; redis: `redis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`
```

File: custom/conf/app.example.ini [hunk 2]
```diff
@@ -1760,7 +1760,7 @@ ROUTER = console
 ;; Provider config options
 ;; memory: doesn't have any config yet
 ;; file: session file path, e.g. `data/sessions`
-;; redis: network=tcp,addr=:6379,password=macaron,db=0,pool_size=100,idle_timeout=180
+;; redis: reddis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s
```
Comment (yardenshoham) [inline custom/conf/app.example.ini]:
```suggestion
;; redis: `redis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`
```

File: custom/conf/app.example.ini [hunk 3]
```diff
@@ -2380,8 +2380,8 @@ ROUTER = console
 ;QUEUE_LENGTH = 1000
 ;;
 ;; Task queue connection string, available only when `QUEUE_TYPE` is `redis`.
-;; If there is a password of redis, use `addrs=127.0.0.1:6379 password=123 db=0`.
-;QUEUE_CONN_STR = "addrs=127.0.0.1:6379 db=0"
+;; If there is a password of redis, use `reddis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`.
+;QUEUE_CONN_STR = "reddis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s"
```
Comment (yardenshoham) [inline custom/conf/app.example.ini]:
```suggestion
;; If there is a password of redis, use `redis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`.
;QUEUE_CONN_STR = "redis://127.0.0.1:6379/0?pool_size=100&idle_timeout=180s"
```

**Review by yardenshoham — CHANGES_REQUESTED** [review]
`reddis` -> `redis`

**Comment by tiny6996** [comment]
thanks for fixing my typos :)

**Review by zeripath — APPROVED** [review]

**Review by KN4CK3R — APPROVED** [review]

**Review by delvh — APPROVED** [review]

**Pull request merged**